appstore/api.py

```diff
@@ -27,6 +27,10 @@ def generate_app_response(results, sort_override=None):
     sorting = sort_override or request.args.get('sort', 'updated')
     if sorting == 'hearts':
         results = results.order_by(App.hearts.desc())
+    elif sorting == 'decaying_hearts':
+        results = results.order_by(App.decaying_hearts.desc())
```

Comment:
Although 'decaying hearts' is a recurring theme in my life, I think that perhaps 'recent_hearts' might be a more pleasant name.

Reply:
Decaying also sounds quite similar to decreasing. Someone might think it's just a fancy way of saying sort from most hearts to fewest hearts.